src/kontrol/deployment.py

```diff
@@ -16,6 +16,7 @@ def __init__(self, name: str, accounts: dict | None = None) -> None:
 
     def generate_header(self) -> list[str]:
```

Comment:
I think it would be nice to have an optional argument to specify the comment you want in the generated file, defaulting to the message you already wrote if no argument is provided